source/ore/routes/list-users.rst

```diff
@@ -0,0 +1,83 @@
+==========
+List Users
+==========
+
+**GET /api/users**
+
+Returns a list of users based on given criteria.
```

Comment:
Well i just dont see a point/usecase in this being exposed. However it does not really matter that much.